packages/web/src/index.tsx

```diff
@@ -3,6 +3,7 @@ import ReactGTM from 'react-gtm-module';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import App from 'components/App';
+// eslint-disable-next-line import/extensions
```

Comment:
This rule may be quite inconvenient for the front-end package

Reply:
Good point, fixed in much more proper way.